fuzz_lightyear/settings.py

```diff
@@ -7,6 +7,15 @@
 class Settings:
     def __init__(self) -> None:
         self.seed = random.getrandbits(128)    # type: int
+        self.unicode_enabled = True            # type: bool
+
+    @property
+    def unicode_enabled(self) -> bool:
+        return self._unicode_enabled
+
+    @unicode_enabled.setter
+    def unicode_enabled(self, value: bool) -> None:
+        self._unicode_enabled = value
```

Comment:
Is there a specific reason why we can't just use the raw value? It doesn't look like the getter or setter is doing anything additional....

Reply:
👍 I guess there's no reason, only to keep it consistent with `seed`. I'll change.